.github/workflows/fetch-telemetry.yml

```
@@ -0,0 +1,59 @@
+          HTTP_STATUS=$(curl -s -o /tmp/overview.json -w "%{http_code}" \
+            --max-time 30 \
+            https://telemetry.cozystack.io/api/overview)
+
+          if [ "$HTTP_STATUS" -ne 200 ]; then
+            echo "Telemetry API returned HTTP $HTTP_STATUS, skipping update"
+            exit 0
+          fi
+
+          # Validate JSON structure (must have generated_at and periods keys)
+          if ! python3 -c "
+import json, sys
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)